server/engine/src/main/java/org/eclipse/lsp/cobol/lsp/SourceUnitGraph.java

```diff
@@ -189,7 +189,7 @@ public boolean isCopybook(String uri) {
               } catch (IOException e) {
                 if (ImplicitCodeUtils.isImplicit(copyUri)) {
```

Comment:
well that was my intention,  to check only in case we are not able to resolve the URI.  since  implicit schema is a thing ,  it won't matter here, but I avoid a ` if` check.

Reply:
Implicit schema is not a thing per RFC 3986 but ImplicitCodeUtils still checks for `implicit://` anyways. Granted, moving it to within the try-block would result in an additional if-statement but this way we are not relying on thrown exceptions to remain the same.

Currently it is getting an IOException by way of the `new URL` portion throwing:
> MalformedURLException – if no protocol is specified, or an unknown protocol is found, or spec is null, or the parsed URL fails to comply with the specific syntax of the associated protocol.

We're just implicitly doing an if-statement check by letting URL handle it for us.

Reply:
removed redundant logger statement